### ISSUE TITLE
Training 3D location regression

### ISSUE THREAD

**Issue by GreatGameDota** [xingyizhou/CenterNet#533]
Do you train a model with the positions encoded as masks? Would they be Gaussian heatmaps or just squares? What loss would you use for that, mse?

**Comment by xingyizhou** [comment]
I didn't understand the idea. I probably have not tried.

**Comment by GreatGameDota** [comment]
> I didn't understand the idea. I probably have not tried.

I'll change the question: What is the strategy for training centernet for regression? For example the dimensions for ddd detection, do you create heatmap masks (and would mse work as the loss function) ?

**Comment by xingyizhou** [comment]
I didn't create masks in an HxW shape, but first extract the peak locations as a vector and create a mask for that fixed-size vector. You can use MSE but I use other losses. Please refer to the paper supplementary for the exact losses.
